File: solutions/cpp/two-fer/1/two_fer.cpp
#include "two_fer.h"
#include<string>
namespace two_fer
{
    std::string two_fer(const std::string n){
        return "One for " + n + ", one for me.";
    }
    std::string two_fer(){
        return "One for you, one for me.";
    }
} // namespace two_fer

File: solutions/java/two-fer/1/src/main/java/Twofer.java
public class Twofer {
    public String twofer(String name) {
    	if(name==null)
			return "One for you, one for me.";
        int lenStr = name.length();
		if(lenStr==0)
			return "One for you, one for me.";
		else
			return "One for " + name + ", one for me.";
    }
}
